modules/client-common/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@aragon/sdk-client-common",
   "author": "Aragon Association",
-  "version": "1.3.0-rc0",
+  "version": "1.4.0-rc0",
```

Comment:
This does not seem correct.
Base support is not a code feature of the SDK as this PR does not introduce nor functionality.  It is a "meta" change. 
So the right number should be 1.3.0-rc0.2 in my opinion to signal that.

Reply:
After discussion, this seems to be correct for the SDK.